crates/node/src/interactive_cli/context.rs

```diff
@@ -165,44 +181,89 @@ impl ContextCommand {
             Commands::Join {
                 private_key,
                 invitation_payload,
+                context,
+                identity,
             } => {
+                
+
                 let response = node
                     .ctx_manager
                     .join_context(private_key, invitation_payload)
                     .await?;
 
-                if let Some((context_id, identity)) = response {
+                if let Some((context_id, public_key)) = response {
+                    // Create identity alias in the context if --name is provided
+                    if let Some(identity) = identity.as_ref() {
+                        if let Err(e) =
+                            node.ctx_manager
+                                .create_alias(*identity, Some(context_id), public_key)
```

Comment:
if the context alias is specified, create a context alias
if the identity alias is specified, create an identity alias inside the context